charts/gardener/provider-local/extension/templates/deployment.yaml

```
@@ -72,6 +75,26 @@ spec:
         - name: IMAGEVECTOR_OVERWRITE
           value: /charts_overwrite/images_overwrite.yaml
         {{- end }}
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: {{ .Values.healthPort }}
+            scheme: HTTP
+          initialDelaySeconds: 30
+          failureThreshold: 5
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 5
```

I suggest the following values (stick to [defaults](https://github.com/kubernetes/kubernetes/blob/release-1.23/pkg/apis/core/v1/defaults.go#L207-L220) for the rest):
```suggestion
          initialDelaySeconds: 10
```